core/arch/arm/plat-rcar/conf.mk

```diff
@@ -44,3 +44,4 @@ $(call force,CFG_ARM32_core,y)
 endif
 
 CFG_WITH_STACK_CANARIES ?= y
+CFG_DT ?= y
```

Comment:
Hi @marex, 

Could you please describe your setup? As far as I know, Renesas's version of ARM-TF does not support DTB. But I'll absolutely love to have this feature.

Also, how this option affects setups where DTB is not present?

Reply:
Mainline TFA does support DT. Re non-DT, there should be no change in behavior.